src/Symfony/Component/HttpKernel/Controller/ArgumentValueResolver/VariadicRequestAttributeResolver.php

```diff
@@ -18,25 +18,27 @@
 /**
  * Grabs the variadic value from the request and returns it.
  *
- * Opposite of {@see ArgumentFromAttribute}.
+ * Opposite of {@see ArgumentFromAttributeResolver}.
```

Comment:
I think the current  way is the correct one. It is supported as soon as the attribute exists. Having the exception in the resolve method makes debugging much easier indeed. The only drawback is that the resolvers must be in the right order.